src/ngAnimate/module.js

```diff
@@ -392,6 +392,93 @@
  *
  * To learn more about what's possible be sure to visit the {@link ngAnimate.$animateCss $animateCss service}.
  *
+ * ## Animation Anchoring (via `ng-animate-ref`)
+ *
+ * ngAnimate in AngularJS 1.4 comes packed with the ability to cross-animate elements between
+ * structural areas of an application (like views) by pairing up elements using an attribute
+ * called `ng-animate-ref`.
+ *
+ * Let's say for example we have two views that are managed by `ng-view` and we want to identify
+ * that there is a relationship between two components situated in different views. By using the
+ * `ng-animate-ref` attribute we can identify that the two components are paired together and we
+ * can then attach an animation between then which is triggered when the view changes.
+ *
+ * ```html
+ * <!-- home.html -->
+ * <a href="#/banner-page">
+ *   <img src="./banner.jpg" ng-animate-ref="banner" class="banner-animation">
+ * </a>
+ *
+ * <!-- banner-page.html -->
+ * <img src="./banner.jpg" ng-animate-ref="banner" class="banner-animation">
+ * ```
+ *
+ * Now, when the view changes (once the link is clicked), ngAnimate will examine the
+ * HTML contents to see if there is a match reference between any components in the view
+ * that is leaving and the view that is entering. It will then attempt to trigger a CSS
+ * animation on the `.banner-animation-anchor` CSS class (notice how `.banner-animation` is
+ * a shared CSS class on both elements containing the `ng-animate-ref` pairing).
+ *
+ * So the two images match since they share the same ref value. They also share a similar CSS
```

Comment:
"So the two images match since they share the same ref value." is not a valid sentence but also does not make sense to me.  Should we just drop the "So"?